app.js

```diff
@@ -39,7 +40,8 @@ app.post( '/', bodyParser.json({limit: '500mb'}), function( req, res ) {
 } );
 
 async function informWatchers( changeSets, res, muCallIdTrail, muSessionId ){
-  services.map( async (entry) => {
+  services.map( async (entry, index) => {
+    entry.index = index;
```

Comment:
this is easy for accessing the index of the entry when building a key to bundle requests